src/cgame/cg_consolecmds.cpp

```diff
@@ -31,27 +31,39 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 /*
 =================
-CG_SizeUp_f
+SizeUp
 
 Keybinding command
 =================
 */
-static void CG_SizeUp_f()
-{
-	trap_Cvar_Set( "cg_viewsize", va( "%i", std::min( cg_viewsize.integer + 10, 100 ) ) );
-}
+class SizeUpCmd : public Cmd::StaticCmd {
+	public:
+		SizeUpCmd(): StaticCmd("sizeup", Cmd::KEY_BINDING, "Scale up the view size") {
```

Comment:
`KEY_BINDING` means a command for managing binds, not a command used as the target of a keybind.